announcements/_posts/2020-02-21-w06-linux-gaming.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Jeffery Russell (@jrtechs)
+title: "Week #6: Linux Gaming"
+layout: post-event
+date-start: "2020-02-21 16:30"
+date-end: "2020-02-21 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey RITluggers!
+
+This week Dev is giving a talk on the exciting developments in Linux gaming. 
+
+After years of anticipation, 
+Linux finally has some traction in the game dev industry. 
+Since 2013, things have only gotten exponentially better for gaming on Linux.
+So what does Linux gaming look like now, 
+and what developments are on the horizon?
+Come to this week's meeting to become an Ebic LINUX Gaimer!
```

Comment:
_oh no_

Reply:
I really don't know how I feel about this sentence lol

Reply:
It's iconic :1st_place_medal: